### PR TITLE
Email Integration For CSSA and enhancement for DERA email functionality

### PR DESCRIPTION
**Pull request by midhun-aot**
# Description

Please provide a summary of the change and the issue fixed. Please include relevant context. List dependency changes.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

## Checklist

- [ ] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have already been accepted and merged

### CODE REVIEW

**Review by nikhila-aot — CHANGES_REQUESTED** [review]

File: forms-flow-ai/forms-flow-ai-ee/forms-flow-bpm/templates/request_updated.ftl [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+</head>
+
+<body style="font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;color: #313132;">
+    <table style="width: 580px;">
+        <tr>
+            <td style="width: 151px;">
+                <img style="width: 151px;" src="https://taft.fin.gov.bc.ca/img/BCID_H_rgb_pos_150.png">
+            </td>
+            <td>
+                <span id="nom_notification" style="overflow: visible;line-height: 30px;text-align: left;font-style: normal;font-weight: bold;font-size: 20px;color: rgba(0, 51, 102, 1);letter-spacing: -0.2px;vertical-align: middle;">Site Remediation System</span>
```
[the remark this answers is not in the document]
Comment (midhun-aot) [reply forms-flow-ai/forms-flow-ai-ee/forms-flow-bpm/templates/request_updated.ftl:18]: done

File: forms-flow-ai/forms-flow-ai-ee/forms-flow-bpm/processes/Bundling/subProcess/CSSABundleWorklow.bpmn [hunk 2]
```diff
@@ -206,22 +215,177 @@ system.out.println ("isSentFromSociety "+ execution.getVariable('isSentFromSocie
...
         <camunda:taskListener event="create">
-          <camunda:script scriptFormat="javascript">task.execution.setVariable('isCssaTeam',false);</camunda:script>
+          <camunda:script scriptFormat="javascript">task.execution.setVariable('isCaseworker',false);</camunda:script>
         </camunda:taskListener>
         <camunda:taskListener class="org.camunda.bpm.extension.hooks.listeners.FormSubmissionListener" event="create" />
         <camunda:taskListener event="complete">
           <camunda:script scriptFormat="javascript">task.execution.setVariable('applicationStatus', task.execution.getVariable('action'));
-task.execution.setVariable('deleteReason', "completed");</camunda:script>
+task.execution.setVariable('deleteReason', "completed");
+task.execution.setVariable('userRole','Case Worker');</camunda:script>
```
[the remark this answers is not in the document]
Comment (midhun-aot) [reply forms-flow-ai/forms-flow-ai-ee/forms-flow-bpm/processes/Bundling/subProcess/CSSABundleWorklow.bpmn:388]: done

File: forms-flow-ai/forms-flow-ai-ee/forms-flow-bpm/processes/Bundling/subProcess/CSSABundleWorklow.bpmn [hunk 3]
```diff
@@ -491,40 +450,51 @@ system.out.println ("Sent to Caseworker");</bpmn:script>
...
             <camunda:script scriptFormat="javascript">task.execution.setVariable('requestStatus', task.execution.getVariable('action'));
-task.execution.setVariable('requestType', 'SRCR');
+task.execution.setVariable('requestType', 'CSSA');
+task.execution.setVariable('cssaStatus', task.execution.getVariable('action'));
+task.execution.setVariable('applicationStatus', task.execution.getVariable('action'));
 task.execution.setVariable('deleteReason', "completed");
-task.execution.setVariable('srcrStatus', task.execution.getVariable('action'));
-task.execution.setVariable('applicationStatus', task.execution.getVariable('action'));</camunda:script>
+task.execution.setVariable('userRole','Statutory Decision Maker');
+task.execution.setVariable('subFormName','DERA Form');</camunda:script>
```
[the remark this answers is not in the document]
Comment (midhun-aot) [reply forms-flow-ai/forms-flow-ai-ee/forms-flow-bpm/processes/Bundling/subProcess/CSSABundleWorklow.bpmn:462]: fixed

File: forms-flow-ai/forms-flow-ai-ee/forms-flow-bpm/processes/Bundling/subProcess/CSSABundleWorklow.bpmn [hunk 4]
```diff
@@ -535,382 +505,288 @@ system.out.println ("Sent to SDM");</bpmn:script>
...
             <camunda:script scriptFormat="javascript">task.execution.setVariable('requestStatus', task.execution.getVariable('action'));
-task.execution.setVariable('requestType', 'SRCR');
+task.execution.setVariable('requestType', 'CSSA');
+task.execution.setVariable('cssaStatus', task.execution.getVariable('action'));
+task.execution.setVariable('applicationStatus', task.execution.getVariable('action'));
 task.execution.setVariable('deleteReason', "completed");
-task.execution.setVariable('srcrStatus', task.execution.getVariable('action'));
-task.execution.setVariable('applicationStatus', task.execution.getVariable('action'));</camunda:script>
+task.execution.setVariable('userRole','Case Worker');
+task.execution.setVariable('subFormName','DERA Form');
```
[the remark this answers is not in the document]
Comment (midhun-aot) [reply forms-flow-ai/forms-flow-ai-ee/forms-flow-bpm/processes/Bundling/subProcess/CSSABundleWorklow.bpmn:518]: fixed

**Review by nikhila-aot — APPROVED** [review]

**Pull request merged**